qemu/tests/rng_hotplug.py

```diff
@@ -122,28 +130,36 @@ def run_subtest(sub_test):
                                        {'id': 'virtio-rng-pci-%d' % num})
             hotplug_rng(vm, new_dev)
```

Comment:
You can invoke simple_hotplug here to support hotplug under q35.  It's ok to send another patch to address this issue, but I'm not sure if below lines would be affected then.

Reply:
This patch is aiming for migration. So I prefer to investigate what you mentioned above later. What's your opinion? @huangyum  Thanks!

Reply:
@ybduan I'm fine with that.